pkg/sessionctx/variable/tidb_vars.go

```diff
@@ -1595,7 +1596,8 @@ var (
 	IgnoreInlistPlanDigest    = atomic.NewBool(DefTiDBIgnoreInlistPlanDigest)
 	TxnEntrySizeLimit         = atomic.NewUint64(DefTiDBTxnEntrySizeLimit)
 
-	SchemaCacheSize = atomic.NewInt64(DefTiDBSchemaCacheSize)
+	SchemaCacheSize           = atomic.NewUint64(DefTiDBSchemaCacheSize)
+	SchemaCacheSizeOriginText = atomic.NewString(strconv.Itoa(DefTiDBSchemaCacheSize))
```

Comment:
We can get the text from SchemaCacheSize like we have done for TiDBServerMemoryLimitSessMinSize